### ISSUE TITLE
Cannot track Auth permission model: psycopg2.errors.UndefinedFunction

### ISSUE THREAD

**Issue by xjlin0** [Opus10/django-pghistory#30]
Thanks for providing such a great tool!  When I tried to track Django 3.2 built-in auth.Permission model:
```python
from django.apps import apps as django_apps

class UsersConfig(AppConfig):
    def ready(self):
        auth_permission_model = django_apps.get_model("auth.Permission", require_ready=False)
        pghistory.track(
            pghistory.Snapshot('permission.snapshot'),
            model_name='PermissionsHistory',
            related_name='history',
            app_label='users',
        )(auth_permission_model)
```
End up with the error when migrate:
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/usr/local/lib/python3.9/site-packages/pgconnection/core.py", line 85, in execute
    return super().execute(sql, args)
psycopg2.errors.UndefinedFunction: function _pgh_attach_context() does not exist
LINE 2: ...NEW."id", NOW(), 'permission.snapshot', NEW."id", _pgh_attac...
                                                             ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
QUERY:  INSERT INTO "users_permissionshistory"
                ("name", "content_type_id", "codename", "id", "pgh_created_at", "pgh_label", "pgh_obj_id", "pgh_context_id") VALUES (NEW."name", NEW."content_type_id", NEW."codename", NEW."id", NOW(), 'permission.snapshot', NEW."id", _pgh_attach_context())
CONTEXT:  PL/pgSQL function pgtrigger_permission_snapshot_insert_839b9() line 14 at SQL statement
```
I am using django-pghistory v1.4.0, Thanks for any suggestions!

**Comment by wesleykendall** [comment]
The pghistory function is made in the `post_migrate` hook, so it looks like there's a chicken and egg problem going on here.

I'm going to address this by either adding the creation of that function to a migration or doing it in a pre-migrate hook. Thanks for reporting!

**Comment by wesleykendall** [comment]
Fixed in version 2.3. The postgres function is now installed in a data migration instead of a post_migrate hook.

New migrations should automatically referenced the latest pghistory  migration as a dependency. Existing migrations should depend on `("pghistory", "0004_auto_20220906_1625")`

**Comment by pgcd** [comment]
I'm experiencing the same problem running tests - as my tests are not configured to run migrations. I'll try to find a way of creating the function independently in unmigrated tests and report back.
